apps/web/.env.example

```
@@ -9,5 +9,4 @@ SUPABASE_URL=
 PLUNK_API_KEY=
 CONTACT_FROM_EMAIL=
 CONTACT_TO_EMAIL=
-ARCJET_KEY=
-NEXT_PUBLIC_TINY_ADZ_SITE_ID=
\ No newline at end of file
+ARCJET_KEY=
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)